azext_iot/tests/iothub/test_iothub_discovery_int.py

```diff
@@ -82,13 +83,13 @@ def test_iothub_targets(self):
         )
         assert_target(desired_target, rg=LIVE_RG, include_events=True)
 
-        sub_targets = discovery.get_targets()
```

Comment:
Why were these assertions commented out?

Reply:
I was testing something, this got in by mistake, will fix.